examples/base_readme.rst

```diff
@@ -95,6 +95,16 @@ your credentials are set up correctly in the default location
     --filename-prefix beyonce_geo \
     --no-print-stream
 
+One or more custom headers can be specified from the command line,
+using the ``--extra-headers`` argument and a JSON-formatted string
```

Comment:
it looks like `--extra-headers` is recommended here, but `--custom-headers` is below. Based on the other edits below it looks "custom" is the language we're using.